internal/release/main.go

```diff
@@ -119,3 +121,27 @@ func generateFile(name, version string) error {
 	jsonEncoder.SetIndent("", "  ")
 	return jsonEncoder.Encode(downloadArchive)
 }
+
+func Builder() *cobra.Command {
+	opts := Opts{}
+	cmd := &cobra.Command{
+		Use:   "main",
+		Short: "Generate the download center json file",
+		Example: `
+  Generate the download center json file for mongocli
+  $ main --version 1.23.0`,
```

Comment:
should we include the (optional) fileName param in this example?

Reply:
Updated